packages/commonwealth/server/migrations/20241114144100-create-referral-link.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  up: async ({ sequelize }) => {
+    await sequelize.transaction(async (transaction) => {
+      await sequelize.query(
+        `ALTER TABLE "Users" ADD COLUMN "referral_link" character varying(255);`,
```

Comment:
Assuming the ref link is unique per user, should there be a unique constraint?

Reply:
it contains a prefix ref_userid_random, so it will be unique by definition... guess we can enforce it at the db level but not necessary